week2/index.js

```diff
@@ -0,0 +1,96 @@
+const fs = require('fs');
+const TODOS_FILENAME = process.env.TODOS_FILENAME || 'todo.json';
+let myTodoList = [];
+
+const readTodosFile = (filename) => {
+    return new Promise((resolve, reject) => {
+        fs.readFile(filename, "utf8", (err, data) => {
+            if (err) {
+                if (err.code === "ENOENT") {
+                    console.log("Sorry, this file doesn't exist.");
+                } else if (err.code === "EACCES") {
+                    console.log("Sorry, but you don't have permission.");
+                } else {
+                    console.log(` unknown error occurred: ${err.message}`);
+                }
+                process.exit();
```

Comment:
You should `reject` the promise here, instead of exiting, and allow the caller to decide what to do with the rejection - exit, or try something else.